projects/workforce-allocations-to-delius/src/main/kotlin/uk/gov/justice/digital/hmpps/api/resource/PersonResource.kt

```diff
@@ -0,0 +1,26 @@
+package uk.gov.justice.digital.hmpps.api.resource
+
+import org.springframework.security.access.prepost.PreAuthorize
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RequestParam
+import org.springframework.web.bind.annotation.RestController
+import uk.gov.justice.digital.hmpps.api.model.Person
+import uk.gov.justice.digital.hmpps.service.PersonService
+
+@RestController
+@RequestMapping("/person")
+class PersonResource(private val personService: PersonService) {
+
+    @PreAuthorize("hasRole('ROLE_ALLOCATION_CONTEXT')")
+    @GetMapping("/{value}")
+    fun findPerson(
+        @PathVariable value: String,
+        @RequestParam(required = false, defaultValue = "CRN") type: IdentifierType
```

Comment:
Should we think about how we want to handle resources with multiple IDs going forward?  Community API uses a path part (e.g. `/person/crn/A123456`), is there a reason for switching to a request parameter here?

I've got no preference either way, but it'd be great to agree on a consistent pattern for these.

A couple of other possible approaches for comparison:
* Named path part: `/person/crn=A123456` and `/person/noms=A1234BC`,
* Request parameter as filter: `/persons?crn=A123456` and `/persons?noms=A1234BC`.